.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
   push:
     branches: [master]
-    tags: ["*"]
   workflow_dispatch:
```

Comment:
Disabling this workflow for pushed tags should resolve the following problem with Codecov:

> Coverage data is based on head [8e9a87f (15 uploads)](https://app.codecov.io/gh/copier-org/copier/commit/8e9a87f79d9a5d6d4ece14fa0c96912ee2373cb2) compared to base [f215a23 (30 uploads)](https://app.codecov.io/gh/copier-org/copier/commit/f215a237441171a84aadc829d695ba3928fc71d9)

30 instead of 15 uploads are cause by the two CI runs triggered by the pushed release commit to `master` and the tag that references that same commit.